ec2snap/ec2snap

```diff
@@ -184,10 +190,7 @@ class ManageSnapshot:
         self._instance_list = instance_list
         self._tags = tags
         self._snapshot_filter = snapshot_filter
-        # receive ["key1:value1", "key2:value2"], transform it to: {key1: value1, key2: value2}
-        self._add_tags = {}
-        if len(add_tags) > 0:
-            self._add_tags = {elem.split(':')[0]: elem.split(':')[1] for elem in add_tags[0]}
```

Comment:
In the class description https://github.com/cycloidio/simple_ec2_snapshot/blob/0edd3aa69af2ecdd5737379f95e04db9d7b2caed/ec2snap/ec2snap#L147

this arg sould be a dict. And we put a list and back magic into the class convert it to a dict.

Just move this magic outside the class. The class now just take a clear dict of tags we want to add. All magic to convert arg from argparser to a dict of tag should be handled outside of this class